lib/selects.bzl

```
@@ -14,6 +14,14 @@
 
 """Skylib module containing convenience interfaces for select()."""
 
+# Target which will always resolve to True. Referenced through a top-level alias
+# so that it will work with WORKSPACE files in bazel.
+_TRUE_TARGET = "@bazel_skylib//lib:always_true"
```

```suggestion
_TRUE_TARGET = Label("//lib:always_true")
```
for compatibility with Bzlmod.

---

Reading up on bzlmod, will update. Thanks for review.